README.md

```diff
@@ -37,171 +37,203 @@ This repository is trying to address those things, using environment variables w
 
 -  Google bucket where terraform state is going to be stored (tf-remote-state)
 
+## Before starting
+### Google Cloud
 
+- Ensure that you've created a service account for you project, following the [guide](https://docs.cloud.google.com/iam/docs/service-accounts-create#creating).
+- Create a set of service account keys, following the [guide](https://docs.cloud.google.com/iam/docs/keys-create-delete#creating).
+	- Note down the location of the service account key for use in the .env file.
+
+### `.env` file
+
+Make the following changes to the `.env` file:
+
+- Set the following variables to the location of the service account keys:
+	- `export SERVICE_ACCOUNT_PKEY="/path/to/sa-pkey.json"` 
+	- `export GOOGLE_APPLICATION_CREDENTIALS="/path/to/sa-pkey.json"`    
+- Set `GNOMAD_DEPLOYMENTS_PROJECT_PATH` to the location that [gnomad-deployments](https://github.com/populationgenomics/gnomad-deployment) was cloned to.
+- Set `GNOMAD_PROJECT_PATH` to the location that the CPG fork of [gnomad browser](https://github.com/populationgenomics/gnomad-browser) was cloned to.
 
 ## Setting Up OurDNA Browser Infrastracture
 
-- Create .env file with all the env variables (look at example.env)
+1. Create .env file with all the env variables (look at `example.env`)
 
-- Create terraform.tfvars in terraform folder (look at terraform.tfvars.example)
+2. Create `terraform.tfvars` in terraform folder (look at `terraform.tfvars.example`)
 
-- Load the environmental variables:
+3. Load the environmental variables:
 ```
 source .env
 ```
 
-- Initialise terraform, provide tf-remote-state bucket on GCP created prior
+4. Initialise terraform:
```

Comment:
make tf-init command will ask for the bucket location where the terraform stet will be stored, please provide tf-remote-state bucket on GCP created prior.